### PR TITLE
[Workspace UI] Color picker and Fill Inspector

### PR DESCRIPTION
**Pull request by michalsek**

### CODE REVIEW

File: packages/noya-react-canvaskit/src/components/Group.native.tsx [hunk 1]
```diff
@@ -96,9 +96,9 @@ const Group: React.FC<PropsWithChildren<GroupProps>> = (props) => {
           layerPaint.setColorFilter(colorFilter);
         }
 
-        if (imageFilter) {
-          layerPaint.setImageFilter(imageFilter);
-        }
+        // if (imageFilter) {
```
Comment (michalsek) [inline packages/noya-react-canvaskit/src/components/Group.native.tsx:99]: add todo

File: packages/noya-designsystem/src/utils/touchable.ts [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+  TapGestureHandlerEventPayload,
+  PanGestureHandlerEventPayload,
+  LongPressGestureHandlerEventPayload,
+} from 'react-native-gesture-handler';
+
+import type { Point } from 'noya-geometry';
+
+/**
+ * Types used by manual gesture utilities
+ */
```
Comment (dabbott) [inline packages/noya-designsystem/src/utils/touchable.ts:15]: Did we take these functions from something open source, or is it all custom? If we took it from somewhere we should add a license/link

**Review by dabbott — APPROVED** [review]
Looks great! I can't wait to try it 🚀

File: packages/noya-designsystem/src/utils/touchable.ts (hunk 2 above)
Comment (michalsek) [reply packages/noya-designsystem/src/utils/touchable.ts:15]: 100% custom :)

**Pull request merged**